### PR TITLE
fix: use new husky install prepare npm script

### PR DESCRIPTION
**Pull request by Uzlopak**
husky does not need the install call, as it seems.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "test": "tap test.js",
-    "prepare": "npx husky install",
```
Comment (Fdawgs) [inline package.json:11]: With husky v9 it just needs `husky`. https://typicode.github.io/husky/how-to.html#manual-setup

Comment (Uzlopak) [reply package.json:11]: Well then my PR is wrong.

Comment (Uzlopak) [reply package.json:11]: @Fdawgs fixed

**Review by Fdawgs — APPROVED** [review]

**Comment by Fdawgs** [comment]
No idea why the CD workflow is unhappy with this. 😠

**Pull request merged**